config/ftbquests/quests/chapters/bigger_reactors.snbt

```diff
@@ -0,0 +1,1084 @@
+				}
+			]
+		}
+		{
+			title: "Expanding Our Reactors"
+			x: 1.5d
+			y: 5.5d
+			description: [
+				"Reactors are multi-block structures, meaning you can create your own custom size!"
+				""
+				"The &lBiggest&r Reactor that you can make is 128x128x192. There are a lot of variables that contribute to the overall output of a reactor, so make sure to experiment!"
+				""
+				"Some overall tips:"
+				""
+				"The taller the reactor, the more fuel it can hold and burn because there are more fuel rods, meaning more overall power. This also means a higher &9burn rate&r."
+				""
+				"The wider the reactor, the more efficient it is. This means less consumption overall."
+			]
+			dependencies: ["4AD8363D7359A072"]
+			min_width: 300
+			id: "3F9D553C9FA64F2A"
+			tasks: [{
+				id: "52294DB4AE793F9C"
+				type: "item"
+				item: "biggerreactors:reactor_casing"
+				count: 16L
+			}]
+			rewards: [
+				{
+					id: "2D71A6EF1CA59FA7"
+					type: "xp"
+					xp: 10
+				}
+				{
+					id: "52181A03434A605B"
+					type: "random"
+					exclude_from_claim_all: true
+					table_id: 487623848494439020L
+				}
+			]
+		}
+		{
+			title: "Turbine Ports"
+			x: -3.0d
+			y: 9.0d
+			shape: "diamond"
+			description: [
+				"To interact with our Turbine, we'll need several &6Ports&r."
+				""
+				"The &9Fluid Port&r is used to insert heated liquid like steam, or extract exhaust, like water. Steam does get condensed back into water, so it's good to have 2 of these for your Turbine: One to input Steam, and one to output Water back into the reactor."
+				""
+				"The &cPower Tap&r extracts power, and is required for us to get power from the turbine."
+				""
+				"You can also make a &eComputer Port&r, but it is completely optional. It allows you to interact with the Turbine with ComputerCraft using scripts."
+			]
+			dependencies: ["4415C9F8DA2D7E68"]
+			id: "186731580B14F9D2"
+			tasks: [
+				{
+					id: "5E9CEEEE330B1DE0"
+					type: "item"
+					item: "biggerreactors:turbine_fluid_port"
+				}
+				{
+					id: "18EB6570007F534A"
+					type: "item"
+					item: "biggerreactors:turbine_power_tap"
+				}
+			]
+			rewards: [
+				{
+					id: "6F76A59F202AD944"
+					type: "item"
+					item: "biggerreactors:turbine_casing"
+					count: 4
+					random_bonus: 4
+				}
+				{
+					id: "7BC028EFCCFAF39A"
+					type: "xp"
+					xp: 10
+				}
+			]
+		}
+		{
+			title: "Creating The Turbine Shaft"
+			x: -4.0d
+			y: 10.0d
+			shape: "diamond"
+			description: [
+				"To get a Turbine spinning, we'll need these &crequired&r components:"
+				""
+				"- &9Rotor Bearings&r are placed at the end of the shaft of the turbine, which needs to be placed in the middle of your turbine. You will need exactly 2 per Turbine, one at the top and one at the base."
+				""
+				"- &eRotor Shafts&r extend from one Rotor Bearing to the other, creating the shaft for the Turbine. The shaft can be vertical or horizontal, but must be in the middle of the turbine."
```

Comment:
It can technically be placed anywhere, center is generally optimal though.

Reply:
I've never seen a Turbine with the Shaft in the middle lol Didn't know that!